packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/error.ts

```diff
@@ -85,17 +85,57 @@ export function handleRegistryErrorResponse({
   };
 }
 
-export function handleHttpErrorResponse({
+export function handleRegistryErrorResponseDr({
```

Comment:
For all of these DR functions before we would only apply the operationOutcome if there were errors. With this new logic now we will always add the operationOutcome.

Reply:
exactly. That way we can actually account for and understand everything we did and did not receive